Bottom: 55576bceeaa70785f02f104788cb3a1e4b1857c1
Top:    3639f29f7e383980f020787070bc490ff47cd5af
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-05 11:39:26 -0500

Refresh of linux

---

diff --git a/.farmrc b/.farmrc
index a1e5b7e..88a03a8 100644
--- a/.farmrc
+++ b/.farmrc
@@ -1,5 +1,6 @@
 farmsrc=$HOME/src/farm
 farmbin=$HOME/src/farm-bin
+linuxgit=$HOME/src/linux
 tags="v2.6.14 v2.6.15 v2.6.16 v2.6.17 v2.6.18 v2.6.19 v2.6.20 v2.6.21
 v2.6.22 v2.6.23 v2.6.24 v2.6.25 v2.6.26 v2.6.27 v2.6.28 v2.6.29 v2.6.30
 v2.6.31 v2.6.32 v2.6.33 v2.6.34 v2.6.35 v2.6.36 v2.6.37 v2.6.38 v2.6.39
diff --git a/farm-checkout b/farm-checkout
index 353f760..7a25464 100755
--- a/farm-checkout
+++ b/farm-checkout
@@ -3,7 +3,7 @@
 set -e
 . $HOME/.farmrc
 
-cd $HOME/src/linux
+cd $linuxgit
 mkdir -p $farmsrc
 for tag in $tags; do
 	srctree=$farmsrc/`echo $tag | sed 's,/,-,'`
